srcpkgs/ghc-bin/template

```diff
@@ -1,8 +1,8 @@
 # Template file for 'ghc-bin'
 pkgname=ghc-bin
 version=8.8.3
-revision=1
-archs="i686 x86_64* ppc64le"
+revision=2
```

Comment:
No need to revbump, it will try to build it if the package isn't available for the specific arch.